src/components/federated-solr-faceted-search.js

```diff
@@ -100,12 +100,13 @@ class FederatedSolrFacetedSearch extends React.Component {
 
     return (
       <LiveAnnouncer>
-        <div className="fs-container">
-          <aside className="fs-aside">
+        <div className={`fs-container ${this.props.options.layoutAndClasses.containerClass}`} style={{ gridTemplateColumns: this.props.options.layoutAndClasses.layout }}>
```

Comment:
We need checks if the the following exist before we use them:
this.props.options.layoutAndClasses.containerClass
this.props.options.layoutAndClasses.layout
this.props.options.layoutAndClasses.asideClass
this.props.options.layoutAndClasses.mainClass

We could use the ternary check like we have on this.props.options.layoutAndClasses.reverseDesktopColumns to be consistent.